analysis/Scripts/Efficiencies/efficiencies.py

```diff
@@ -137,32 +158,40 @@ def main(argv):
 					run = 2
 					cuts = Imports.getDataCuts(run)
 					turbo = "lcplus_Hlt1TrackMVADecision_TOS==1"
-				
+
 				#WHAT DO I NEED FOR TRIGGER CUTS?!!
```

Comment:
L0: L0Hadron_TOS
Hlt1 (run1): Hlt1TrackAllL0Decision_TOS
Hlt1 (run2): Hlt1TrackMVADecision_TOS
Hlt2 (run1): Hlt2CharmHadD2HHHDecision (or Hlt2CharmHadLambdaC2KPPiDecision ?)
Hlt2 (run2, Lc): Hlt2CharmHadLcpToPpKmPipTurbo
Hlt2 (run2, Xic): Hlt2CharmHadXicpToPpKmPipTurbo

Reply:
This is fixed, the new cuts can be found in the Imports library. For the fact that this is in the code, I will fix all this just after the merge, when I dive back into efficiencies